app/assets/stylesheets/modules/crowd-map-slider.css.scss

```diff
@@ -131,3 +135,71 @@ input[type="range"]:focus::-ms-fill-upper {
     }
   }
 }
+
+.zoom-slider-container {
```

Comment:
@informatykgosia could you please extract all styles for the zoom slider to a seperate file?